dysnix/op-node/values.yaml

```diff
@@ -0,0 +1,162 @@
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+  # runAsGroup: 1000
+
+## Override op-node command (can be templated)
+command: []
+
+## Extra op-node arguments (can be templated)
+extraArgs: []
+
+## Extra init containers, can be templated
+initContainers: []
+
+## Sidecar containers, can be templated
+sidecarContainers: []
+
+## Services config
+services:
+  p2p:
+    enabled: true
+    type: NodePort
+    loadBalancerIP: ""
+    port: 9222
+    # it's better to set nodePort equal to .Values.config.node.p2p.port when the svc type is "NodePort"
+    # nodePort: 9222
+    annotations: {}
+    publishNotReadyAddresses: true
+  p2pDiscovery:
+    enabled: true
+    type: NodePort
+    loadBalancerIP: ""
+    port: 9222
+    # it's better to set nodePort equal to .Values.config.node.p2p.port when the svc type is "NodePort"
+    # nodePort: 9222
+    annotations: {}
+    publishNotReadyAddresses: true
+  rpc:
+    enabled: true
+    type: ClusterIP
+    port: 9545
+    annotations: {}
+  metrics:
+    enabled: true
+    type: ClusterIP
+    port: 7300
+    annotations: {}
+    publishNotReadyAddresses: true
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    # cert-manager.io/cluster-issuer: letsencrypt-prod
+  hosts: []
+    # - host: op-node.examplerpc.com
+    #   paths:
+    #     - path: /
+    #       pathType: ImplementationSpecific
+  tls: []
+    # - secretName: op-node-tls
+    #   hosts:
+    #     - op-node.examplerpc.com
+
+# Create Prometheus Operator serviceMonitor
+serviceMonitor:
+  enabled: false
+  # interval: 10s
+  # scrapeTimeout: 2s
+  # honorLabels: true
+  # relabelings: []
+  # metricRelabelings: []
+
+livenessProbe:
+  enabled: false
+  # initialDelaySeconds: 60
+  # periodSeconds: 30
+  # successThreshold: 1
+  # failureThreshold: 3
+  # timeoutSeconds: 5
+  # httpGet:
+  #   path: /healthz
+  #   port: rpc
+
+readinessProbe:
+  enabled: false
+  # initialDelaySeconds: 60
+  # periodSeconds: 30
+  # successThreshold: 1
+  # failureThreshold: 3
+  # timeoutSeconds: 5
+  # httpGet:
+  #   path: /healthz
+  #   port: rpc
+
+resources: {}
+
+affinity: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+## Main op-node config
+config:
+  jwt: ""                              # REQUIRED for communication with op-geth
+  network: op-mainnet                  # which network to use
+  port: 9545
+  logLevel: INFO
+  l1:
+    url: https://1rpc.io/eth           # eth L1 node rpc url, can also be ws://
+    trustrpc: false                    # enable if you trust L1 provider and want to fetch data from it faster
+    rpckind: standard                  # can be: alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon, basic, any, standard
+  l2:
+    url: http://op-geth.optimism:8551  # authrpc addr of op-geth L2 node, can also be ws://
```

Comment:
Is it a hardcode to `optimism` helm release namespace? WDYT on omitting the namespace by default?

Reply:
We can do that, but currently we are expecting for user to change l2.url anyways to meet op-geth release name and namespace properly.